### PR TITLE
fix(episode-item): show title and subtile also on mobile devices

### LINKED ISSUE #987: No text on episodes list of a show on mobile
**Describe The Bug**
On mobile, there are no episode titles or description or nothing on a show's page.

**Steps To Reproduce**

1. Open a show page on short width screen

**Expected Behavior**

Have at least the number and title of the episodes, maybe part of or the whole description?

**Screenshots**

### PR DESCRIPTION
**Pull request by MarcelCoding**
Also show the the episode title also on mobile devices.
fixes #987

### CODE REVIEW

**Comment by ThibaultNocchi** [comment]
For the record this looks like it on 1080x1920
Screenshot_2021-04-29 Jellyfin

File: client/components/Item/SeasonTabs.vue [hunk 1]
```diff
@@ -88,3 +91,9 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style lang="scss">
```
Comment (ThibaultNocchi) [inline client/components/Item/SeasonTabs.vue:95]: I guess this should be scoped? :)
```suggestion
<style scoped lang="scss">
```

**Review by ThibaultNocchi — COMMENTED** [review]
For the others this looks like this on 1080p width

**Comment by MarcelCoding** [comment]
I'll change it.

**Comment by ThibaultNocchi** [comment]
Way clearer :) I'd just add a small margin below the texts to better separate the episodes and quickly see to which episode the text is referencing

print_screen_2021-04-29-15-06-04

**Comment by MarcelCoding** [comment]
image
Like this?

**Comment by ThibaultNocchi** [comment]
Looks a lot better, I'll let the others give their opinion :)
print_screen_2021-04-29-18-10-28

**Review by ferferga — APPROVED** [review]
LGTM

**Review by ThibaultNocchi — APPROVED** [review]

**Pull request merged**